_site.yml

```diff
@@ -28,7 +28,7 @@ collections:
     subscribe: _subscribe.html
 output: distill::distill_article
 rss:
-  full_content: TRUE
+  full_content: true
```

Comment:
https://rstudio.github.io/distill/blog.html says `true` instead of `TRUE` :slightly_smiling_face: